### ISSUE TITLE
Add methods track_download_by_url and track_info

### ISSUE THREAD

**Issue by hittiks** [adw0rd/instagrapi#709]
So I propose (I test this code, and it's work):

```
def track_download_by_url(self, url: str, filename: str = "", folder: Path = "") -> Path:
    fname = urlparse(url).path.rsplit("/", 1)[1].strip()
    assert fname, """The URL must contain the path to the file (m4a or mp3)."""
    filename = "%s.%s" % (filename, fname.rsplit(".", 1)[1]) if filename else fname
    path = Path(folder) / filename
    response = requests.get(url, stream=True)
    response.raise_for_status()
    with open(path, "wb") as f:
        response.raw.decode_content = True
        shutil.copyfileobj(response.raw, f)
    return path.resolve()
```

Example url:
`https://scontent.fdnk3-1.fna.fbcdn.net/v/t39.12897-6/280752329_537789804610385_4797754427131570699_n.m4a?_nc_cat=1&ccb=1-7&_nc_sid=02c1ff&_nc_ohc=7_-yY46rMD4AX8haatv&_nc_ad=z-m&_nc_cid=1489&_nc_ht=scontent.fdnk3-1.fna&oh=00_AT88pUY8Qt6iRFykFG-DAcEZxuzgwGnZPV8Mjs0M3W8nvA&oe=62A80E3F`

But have no idea for something like https://www.instagram.com/reels/audio/698069981278947/

**Comment by adw0rd** [comment]
@hittiks done `instagrapi 1.16.28`

https://adw0rd.github.io/instagrapi/usage-guide/track.html

**Comment by hittiks** [comment]
Thanks for this functionality, but I meant that I needed to download the track from the link to it, and not to reels.
However, I have already solved this problem myself by simply replacing some parameters in the data parameter.

So, with link: https://www.instagram.com/reels/audio/698069981278947/
We have this code:
`    data = {
            "tab_type": "clips",
            "referrer_media_id": "",
            "_uuid": self.uuid,
            "audio_cluster_id": 698069981278947,
            "original_sound_audio_asset_id": 698069981278947,
        }

    result = self.private_request("clips/music/", data)`

And in this case I don't need to get the music_canonical_id somehow.

At the end, I got a tool for downloading audio (track) from a link to it :)
